scripts/run_generate_graphs

```diff
@@ -0,0 +1,14 @@
+#!/usr/bin/env python3
+import os
+import json
+
+from generate_graphs import generate_graph
+
+if __name__ == "__main__":
+    root = os.path.dirname(os.path.realpath(__file__))
+    with open('./static/repos.json', 'r') as f:
+        repos = json.load(f)
+    generate_graph(repos, "odin", "./static/odin-graph.json")
```

Comment:
do you think it's worth doing this by the sprawling config object (so that it's easy to add more later etc)?